.github/workflows/build_with_conda.yml

```diff
@@ -108,8 +109,9 @@ jobs:
       ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true
     steps:
       - uses: actions/checkout@v3.3.0
-        with:
-          submodules: recursive
+        # DONT use recursive submodules checkout to simulate conda feedstock build
+        # with:
```

Comment:
Delete lines 113 and 114

Reply:
I prefer to leave them in, I think that it makes it clearer why we are not doing the recursive checkout